docs/logs/correlation/README.md

```diff
@@ -0,0 +1,10 @@
+# Correlate Logs with Traces
+
+Logs from `ILogger` can be correlated with distributed tracing by enriching each
```

Comment:
should we call out that this works only for 5.0 or above?

Reply:
Good point, let me add it.